lib/routes/routeBackbeat.js

```diff
@@ -605,7 +605,7 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
                     // transition processor.
                     omVal.location && Array.isArray(omVal.location) &&
                     locationKeysHaveChanged(objMd.location, omVal.location)) {
-                    log.info('removing old data locations', {
+                    log.debug('removing old data locations', {
```

Comment:
i think this specific case is important to keep with a higher log level : this is a case where we actually remove _data_ on a `putMetadata` call...
(though it should be noted that in Zenko it goes through `internal-cloudserver`, which does has lower verbosity by default and does not log `info` level...)

Reply:
I don't think it is a warning, since we are expecting it to remove data: so in a sense WARN is too alarmist...

Increasing the log level allows seeing these in `internal-cloudserver`, but I am not sure we really _need_ to see everyone of these in general: it will happen for every transition or restore expiration, nothing significant from a system point of view.

Basically i think that it can be confusing that data gets deleted on a `putMetadata`, and esp. counter-intuitive: so should not require too much verbosity to surface, when people start looking for it.

Reply:
Makes sense, I am reverting as info. I would say that having more log in internal cloudserver would be great anyway, at the very least, to know what requests are being processed (not only when there is an error), but at the same time, it leads to way less logs, so higher chance to keep all relevant error in the reports...

Reply:
Done, closing this comment.